bin/run-tests

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 set -e
```

Comment:
Completely unrelated, I can't find -e option in the [manual](http://man7.org/linux/man-pages/man1/bash.1.html#OPTIONS), does anyone know it's purpose?

Reply:
http://www.davidpashley.com/articles/writing-robust-shell-scripts/#id2382181

Reply:
Thanks